web-api/src/v2/getDocumentDownloadUrlLambda.test.js

```diff
@@ -0,0 +1,236 @@
+const createApplicationContext = require('../applicationContext');
+const {
+  getDocumentDownloadUrlLambda,
+} = require('./getDocumentDownloadUrlLambda');
+const { MOCK_USERS } = require('../../../shared/src/test/mockUsers');
+
+const REQUEST_EVENT = {
+  body: {},
+  headers: {},
+  path: '',
+  pathParameters: {},
+  queryStringParameters: {},
+};
+
+const createSilentAppContext = user => {
```

Comment:
refactoring nit: pull this out into its own helper

Reply:
added to trello backlog